refresh/templates/services/watsonconversation/initializeBluemixService.swift

```diff
@@ -1,3 +1,5 @@
     let service = try manager.getWatsonConversationService(name: "<%- service.name %>")
 
-    conversation = Conversation(service: service)
\ No newline at end of file
+    let version = "<%- service.version %>"
+
+    conversation = Conversation(service: service, version: version)
```

Comment:
I think it would be better not to introduce a new local here. Instead we could do:
`conversation = Conversation(service: service, version: "<%- service.version %>")`